airflow/sentry.py

```diff
@@ -55,9 +55,6 @@ def flush(self):
 Sentry: DummySentry = DummySentry()
 if conf.getboolean("sentry", "sentry_on", fallback=False):
     import sentry_sdk
-
-    # Verify blinker installation
-    from blinker import signal  # noqa: F401
```

Comment:
I spend a quite few time to find out why we need this dependency 🤣 because it is only use in this place in entire codebase. However for other PR found that we use in past some flask signals (maybe also now who knows).
So as soon as we could bump to Flask 2.3 (Connexion 3 and latest FAB required) we might remove it from core.